### PR TITLE
Ensure correct theme

### PR DESCRIPTION
**Pull request by sawirricardo**
Continued from https://github.com/filamentphp/filament/pull/7217

- [x] Changes have been thoroughly tested to not break existing functionality.
- [x] New functionality has been documented or existing documentation has been updated to reflect changes.
- [x] Visual changes are explained in the PR description using a screenshot/recording of before and after.

### CODE REVIEW

File: packages/panels/resources/views/components/layout/base.blade.php [hunk 1]
```diff
@@ -78,6 +78,16 @@
...
         @endif
 
+        @if (! filament()->hasDarkMode())
+            <script>
+                const theme = localStorage.getItem('theme') ?? 'system'
+                
+                if (theme != 'light') {
+                    localStorage.setItem('theme', 'light')
+                }
+            </script>
```
Comment (zepfietje) [inline packages/panels/resources/views/components/layout/base.blade.php]: We just need `localStorage.setItem('theme', 'light')`, right?

Comment (sawirricardo) [reply packages/panels/resources/views/components/layout/base.blade.php]: That's correct, too

Comment (sawirricardo) [reply packages/panels/resources/views/components/layout/base.blade.php]: https://github.com/filamentphp/filament/pull/7224/commits/eabd152d81ea6cf7494326295587662414f5ab45

**Comment by zepfietje** [comment]
Thanks

**Comment by sawirricardo** [comment]
nice! thanks @zepfietje

**Pull request merged**